components/database/mongodb-40/Makefile

```diff
@@ -27,7 +27,7 @@ COMPONENT_CLASSIFICATION= System/Databases
 COMPONENT_SRC=		$(COMPONENT_NAME)-src-r$(COMPONENT_VERSION)
 COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.gz
 COMPONENT_ARCHIVE_URL=	https://fastdl.mongodb.org/src/$(COMPONENT_ARCHIVE)
-COMPONENT_ARCHIVE_HASH=	sha256:fbbe840e62376fe850775e98eb10fdf40594a023ecf308abec6dcec44d2bce0c
+COMPONENT_ARCHIVE_HASH=	sha256:c2672443f01ea4523a356edc8804cb6af31e831f63a88ee4a100aafbff443e4f
 COMPONENT_LICENSE= AGPLv3, zlib, Apache v2.0
```

Comment:
COMPONENT_LICENSE has changed

Reply:
I have installed this packages and AFAIS python-27 is used.
library/python/pip-27
library/python/requests-27
library/python/pip-35
Do all tests fail?